### ISSUE TITLE
DeepLearning_ImageClassification_TensorFlow sample in 0.11

### ISSUE THREAD

**Issue by bandozia** [dotnet/machinelearning-samples#325]
I'm struggling to adapt this example to version 0.11
In TFModelScorer.LoadModel,
MLContext.Transforms.Resize and ImagePixelExtractorTransformer are not available, so I tried to do it this way:

Change
```csharp
 var pipeline = mlContext.Transforms.LoadImages(imageFolder: imagesFolder, columns: (outputColumnName: ImageReal, inputColumnName: nameof(ImageNetData.ImagePath)))
                            .Append(mlContext.Transforms.Resize(outputColumnName: ImageReal, imageWidth: ImageNetSettings.imageWidth, imageHeight: ImageNetSettings.imageHeight, inputColumnName: ImageReal))
                            .Append(mlContext.Transforms.ExtractPixels(columns: new[] { new ImagePixelExtractorTransformer.ColumnInfo(name: InceptionSettings.inputTensorName, inputColumnName: ImageReal, interleave: ImageNetSettings.channelsLast, offset: ImageNetSettings.mean) }))
                            .Append(mlContext.Transforms.ScoreTensorFlowModel(modelLocation:modelLocation, outputColumnNames:new[] { InceptionSettings.outputTensorName }, inputColumnNames: new[] { InceptionSettings.inputTensorName } ));
```
to 

```csharp
var pipeline = mlContext.Transforms.LoadImages(imageFolder: imagesFolder, (outputColumnName: ImageReal, inputColumnName: nameof(ImageNetData.ImagePath)))
                .Append(mlContext.Transforms.ResizeImages(outputColumnName: ImageReal, imageWidth: ImageNetSettings.imageWidth, imageHeight: ImageNetSettings.imageHeight, inputColumnName: ImageReal))
                .Append(mlContext.Transforms.ExtractPixels(columns: new[] { new ImagePixelExtractingEstimator.ColumnOptions(name: InceptionSettings.inputTensorName, inputColumnName: ImageReal, interleave: ImageNetSettings.channelsLast, offset: ImageNetSettings.mean) }))
                .Append(mlContext.Transforms.ScoreTensorFlowModel(modelLocation: modelLocation, outputColumnNames: new[] { InceptionSettings.outputTensorName }, inputColumnNames: new[] { InceptionSettings.inputTensorName }));
```
but this results in an error, It seems like I'm missing a dimension

> input must be 4-dimensional[224,224,3]
         [[{{node conv2d0_pre_relu/conv}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], padding="SAME", strides=[1, 2, 2, 1], use_cudnn_on_gpu=true, _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_input_0_0, conv2d0_w)]]

Since the tensorflow model is the same as it works in v0.10,  I must be doing something wrong.
Does anyone know the proper way to adapt this example?

**Comment by CESARDELATORRE** [comment]
@bandozia - TensorFlow support in 0.11 is broken. You can:
- 0.10 
- Wait for 0.12 to be released at NuGet in a few days
- Use 0.12 daily-drops-preview available at MyGet

See issue which is already closed since it's been fixed for 0.12:
https://github.com/dotnet/machinelearning/issues/2778 

Hope it helps!

**Comment by prathyusha12345** [comment]
TenosrFlow samples are working with ML.Net 10.0-preview. we migrated the samples. Please try to follow the code in those samples. For example: https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow.

Closing this issue. thanks 👍
